src/argilla/client/workspaces.py

```diff
@@ -262,7 +262,7 @@ def __active_client() -> "httpx.Client":
             raise RuntimeError(f"The `rg.active_client()` is not available or not respoding.") from e
 
     @classmethod
-    def __new_instance(
+    def _new_instance(
```

Comment:
Why is this change? Is there any strong reason for it?

Reply:
we need to create workspaces in unit tests, a there is no easy way since the `__init__` cannot be used.